puzzles/templatetags/puzzle_extras.py

```diff
@@ -72,21 +74,22 @@ def __get_puzzle_class(sorted_nodes):
     puzzle_class = __get_puzzle_class(sorted_nodes)
 
     context = {
-        'rows': zip(sorted_puzzles, status_forms, meta_forms, edit_forms, tag_forms, puzzle_class),
+        'rows': zip(sorted_puzzles, active_users, status_forms, meta_forms, edit_forms, tag_forms, puzzle_class),
         'guess_form': AnswerForm(),
         'slack_base_url': settings.SLACK_BASE_URL,
     }
     return context
 
 @register.inclusion_tag('title.html')
-def get_title(puzzle, edit_form):
+def get_title(puzzle, active_users, edit_form):
     badge = ''
     if puzzle.is_meta:
         badge = 'META'
     context = {
         'puzzle': puzzle,
+        'active_users': active_users,
```

Comment:
why not puzzle.active_users.all() here?

Reply:
Hmm, that might work, too. Let me try it out.

Reply:
It worked! I updated my PR.